src/dynarec/la64/dynarec_la64_660f.c

```diff
@@ -316,6 +316,81 @@ uintptr_t dynarec64_660F(dynarec_la64_t* dyn, uintptr_t addr, uintptr_t ip, int
                     GETEX(q1, 0, 0);
                     VSIGNCOV_W(q0, q1, q0);
                     break;
+                case 0x0B:
+                    INST_NAME("PMULHRSW Gx,Ex");
+                    nextop = F8;
+                    GETGX(q0, 1);
+                    GETEX(q1, 0, 0);
+                    v0 = fpu_get_scratch(dyn);
+                    v1 = fpu_get_scratch(dyn);
+                    VMULWEV_W_H(v0, q0, q1);
```

Comment:
To be honest, this seems a bit too much of instructions to implement this, maybe `xvexth.w.h` is helpful here?

Reply:
LASX is there, so it's a waste to have the upper 128bit does nothing :)

Reply:
`EV` in `VMULWEV_W_H` means Even digit and `OD` means Odd digit, so there might not be a chance to move v1 into v0's upper 128bit:
```
VMULWEV_W_H(v0, q0, q1);
VMULWOD_W_H(v1, q0, q1);
XVEXTH_W_H(v0, v1); 
XVSRAI_W(v0, v0, 14); 
XVADDI_WU(v0, v0, 1);
XVSRANI_H_W(q0, v0, 1);
```

Please point out my fault.

Thanks,
Leslie Zhai

Reply:
```c
VEXT2XV_W_H(v0, q0);
VEXT2XV_W_H(v1, q1);
XVMUL_W(v0, v0, v1);
XVSRLI_W(v0, v0, 14);
XVADDI_WU(v0, v0, 1);
XVSRLNI_H_W(v0, v0, v0);
XVPICKOD_D(v0, v0, v0);
```

Not sure if this is correct, but the idea here is to use the high 128 bit of LASX.

Reply:
Or maybe this to save 1 more instruction? Like I said, I'm not sure, but I do think it's doable.

```c
VEXT2XV_W_H(v0, q0);
VEXT2XV_W_H(v1, q1);
XVMUL_W(v0, v0, v1);
XVSRLNI_H_W(v0, v0, 14);
XVADDI_WU(v0, v0, 1);
XVPICKOD_D(v0, v0, v0);
```

Reply:
typo?
```
-XVPICKOD_D(v0, v0, v0);
+XVPICKOD_D(q0, v0, v0);
```

sse_intrinsics pmulhrsw failed.